ipc/common/arguments.go

```diff
@@ -18,14 +19,12 @@ type ArwenArguments struct {
 
 // VMHostArguments represents the arguments to be passed to VMHost
 type VMHostArguments struct {
-	VMType        []byte
-	BlockGasLimit uint64
-	GasSchedule   GasScheduleMap
+	VMType                    []byte
```

Comment:
The `VMHostArguments` structure could be moved in `arwen/common.go`, if possible, and then `NewArwenVM()` can be changed to take it as argument, instead of the individual values.

Reply:
Fixed (and renamed).